src/main/kotlin/com/plus/taxiapp/infra/middleware/validation/ValidationMiddleWare.kt

```diff
@@ -0,0 +1,17 @@
+package com.plus.taxiapp.infra.middleware.validation
+
+import org.springframework.stereotype.Component
+import java.util.logging.Logger
+
+@Component
+class ValidationMiddleWare(
+    private val logger: Logger = Logger.getLogger("Validation Middle Ware"),
+) {
+    fun validationAccount(accountNum: String, accountPassword: String) {
+        logger.info("Validation Account")
+    }
+
+    fun validationCard(cardNum: String, cardPassword: String, cvc: Int, expiredDate: String) {
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline